content/en/integrations/guide/azure-manual-setup.md

```diff
@@ -135,6 +135,49 @@ azure role assignment create --objectId <OBJECT_ID> -o "Monitoring Reader" -c /s
 
 ### Integrating through the Azure portal
 
+{{< tabs >}}
+{{% tab "ARM template" %}}
+
+1. In the Azure integration tile, select **Configuration** > **New App Registration** > **Using Azure Portal**.
+
+2. Select Management Group (Auto-Discover) or Individual Subscriptions. If you select Management Group, Datadog automatically discovers and monitors any subscriptions within that selected scope, including any subscriptions created in the future. You must have the owner role in the Management Group selected. If you select to monitor individual subscriptions, you must have the owner role in any subscriptions you wish to monitor.
+
+3. Click **Open Template**.
+
+{{< img src="integrations/guide/azure_manual_setup/azure_tile_arm_template.png" alt="The Azure tile in the Datadog integrations page with Using Azure Portal and Management Group selected" popup="true" style="width:80%;" >}}
+
+4. Select the region, subscription, and resource group for the template to be deployed.
+
+   **Note**: The selection of region, subscription, and resource group only defines where this template is deployed. It has no impact on which subscriptions are monitored by Datadog.
+
+5. Click **Next**.
+
+6. Select the option to create a new service principal type, and click the **Change selection** link for service principal. A form to create a new app registration appears on the right.
+
+{{< img src="integrations/guide/azure_manual_setup/arm_template_service_principal.png" alt="The service principal page in the Azure ARM template with the option to Create New selected and the link to Change Selection highlighted" popup="true" style="width:80%;" >}}
+
+7. Enter a name for the app registration, select the supported account types, and click **Register**.
+
+8. A page opens to create a client secret. Click **+ New client secret** to add a client secret.
+
+9. Copy the value of the client secret, and click the close **(X)** button in the top-right corner of the screen.
+
+10. Paste the value of the client secret in the corresponding field on the template, and click **Next**.
+
+11. Provide a Datadog API key and Datadog Application key value in the corresponding fields. If you launched the template from the Azure integration page in Datadog, you can copy the keys provided there. Otherwise, you can find your API and App keys in the Access section of the Organization Settings.
+
+   **Note**: If you've selected to monitor individual subscriptions rather than a management group, select the subscriptions to monitor from the **Subscriptions to monitor** dropdown.
+
+12. Select your Datadog site, as well as any other integration configuration options, such as host filters and whether to collect resources for [Cloud Security Management][17].
+
+13. Click **Next** or **Review + create**, then click **Create**.
```

Comment:
What is the difference between Next or Review + create? We should probably just list the happy path we want users to take, but I may be missing context.

Reply:
Thanks for flagging, there is some redundancy in the Azure UI here ([screenshot](https://a.cl.ly/L1uyo9ov)). You make a good point about keeping it simple, updated in [c0b67e1](https://github.com/DataDog/documentation/pull/21084/commits/c0b67e12745881e3395854c6635784953e219c84)